aesm-client/src/imp/unix.rs

```diff
@@ -149,10 +150,13 @@ impl AesmClient {
     /// Obtain launch token
     pub fn get_launch_token(
         &self,
-        mr_enclave: Vec<u8>,
-        signer_modulus: Vec<u8>,
-        attributes: Vec<u8>,
+        sigstruct: &Sigstruct,
+        attributes: Attributes,
     ) -> Result<Vec<u8>> {
+        let mr_enclave: Vec<u8> = sigstruct.enclavehash.to_vec();
+        let signer_modulus: Vec<u8> = sigstruct.modulus.to_vec();
+        let attributes: Vec<u8> = attributes.as_ref().to_vec();
```

Comment:
Type annotations are unnecessary, but also you can just write `sigstruct.enclavehash.to_vec()` etc. directly where it's being used below.